### DISCUSSION TITLE
rmw v0.8.0 released

### DISCUSSION THREAD

**Discussion by andy5995** [theimpossibleastronaut/rmw#322] [Announcements]
* The config option 'purge_after' has been deprecated and replaced
  with 'expire_age' ('purge_after' will still be accepted, but the user
  will be given a deprecation warning).

  * When '-u' is used, the mrl file will no longer be removed, but
  emptied instead. If a user gives the '-u' option when the mrl file is
  empty, they will get a friendlier message stating that there are no
  items in the list (instead of an error message about the file not
  being found).

  * fixed handling of dot dirs ('.' and '..'). Instead of giving an
  error message and quitting, rmw will print a message stating that
  they'll be skipped.

  * The environmental variable RMWTEST_HOME has been replaced with
  RMW_FAKE_HOME (Trying to use RMWTEST_HOME will give a deprecation
  warning and still work).

  * Expanded the man page and updated the README (improved documentation)

  * Replaced autotools build system with meson

  * rmw now uses the canfigger library for parsing the configuration file.

  - disabled a restore test that failed when 'make check' is run with
  superuser privileges

[![Packaging status](https://repology.org/badge/vertical-allrepos/rmw.svg)](https://repology.org/project/rmw/versions)

## sha256sum
```
17b11b771befb89adbd7764fb897dff1a32eed34f8cd4d8b495da7f2a66de3b8  rmw-0.8.0.tar.gz
b1766452b76a3736ccb03d45769fccdd5228d85b18476fb002d40f847b0c1333  rmw_0.8.0_amd64.deb
6657b7b448822aa1bdaa8bd636833cece71bfa99014141a55618f31750963f7f  rmw-0.8.0-x86_64.AppImage
```

<hr /><em>This discussion was created from the release <a href='https://github.com/theimpossibleastronaut/rmw/releases/tag/v0.8.0'>rmw v0.8.0 released</a>.</em>

**Comment by andy5995** [comment]
I may have created the AppImage just a little incorrectly when I released it earlier today. I've recreated the AppImage and replaced the one that was released. I've updated the sha256sum accordingly.

The sum for the AppImage that was originally released is `fa2b23247a889704cdc5250dc0620b0b1542f2d3b053157e5d237dbb029f88d8`. There's no security issue, but when using `linuxdeploy`, I used `--executable /usr/bin/rmw` instead of `--executable AppDir/usr/bin/rmw`.

**Comment by martijndeb** [comment]
Congratulations! \o/

**Comment by andy5995** [comment]
I've replaced the AppImage one more time, so it will pass the test on https://github.com/AppImage/appimage.github.io (it needed to be built on an older system)

6ddc12d57ce254b6429b4b438d0d7cc8b2c37108c9d1c0717b98dba782a34733  rmw-0.8.0-x86_64.AppImage

**Comment by andy5995** [comment]
I've removed `rmw-0.8.0-x86_64.AppImage` in favor of `rmw-0.8.0-2-x86_64.AppImage`.

783d2f88fa8184665f13b80bc030852ff206e97e2ac7a80f53d125816b04a381  rmw-0.8.0-2-x86_64.AppImage
